proofs/cbmc/shake128_squeezeblocks/Makefile

```diff
@@ -0,0 +1,54 @@
+# SPDX-License-Identifier: Apache-2.0
+
+include ../Makefile_params.common
+
+HARNESS_ENTRY = harness
+HARNESS_FILE = shake128_squeezeblocks_harness
+
+# This should be a unique identifier for this proof, and will appear on the
+# Litani dashboard. It can be human-readable and contain spaces if you wish.
+PROOF_UID = shake128_squeezeblocks
+
+DEFINES +=
+INCLUDES +=
+
+REMOVE_FUNCTION_BODY +=
+UNWINDSET +=
+
+PROOF_SOURCES += $(PROOFDIR)/$(HARNESS_FILE).c
+PROJECT_SOURCES += $(SRCDIR)/mldsa/fips202/fips202.c
+
+CHECK_FUNCTION_CONTRACTS=$(FIPS202_NAMESPACE)shake128_squeezeblocks
+USE_FUNCTION_CONTRACTS=keccak_squeezeblocks
```

Comment:
This contract does not exist yet in this branch. This PR should be rebased on top of #187.

Reply:
Yes, sorry, was meaning to put depends on https://github.com/pq-code-package/mldsa-native/pull/187 in the description!

Reply:
Thanks. You can already rebase it on top of #187 if you want. Then we can see that it works, review it, and approve it so that it can be merged right after #187.